content/installation/java/installation/MavenCargoPlugin.md

```diff
@@ -5,27 +5,25 @@ tags: "java agent installation maven codehaus"
 -->
 
 
-## Prerequisites
-Download the Contrast plugin (***contrast.jar***) from the Contrast site.
-
 ## Instructions
 
-* Add a ```<cargo.jvmargs>``` property to the ```<properties>``` section of ```<configuration>``` in your ***pom.xml***, like this: </configuration 
+* Add a `<cargo.jvmargs>` property to the `<properties>` section of `<configuration>` in your ***pom.xml***, like this:
 
 ````
 <build>
   <plugins>
     <plugin>
-      <groupId>org.codehaus.cargo.maven2</groupId>
+      <groupId>org.codehaus.cargo</groupId>
       <artifactId>cargo-maven2-plugin</artifactId>
+      <version>1.7.3</version>
       <configuration>
         <container>
-          <containerId>tomcat5x</containerId>
+          <containerId>jetty9x</containerId>
           [...]
         </container>
         <configuration>
           <properties>
-            <cargo.jvmargs>-javaagent:${DOWNLOADS}/contrast.jar -noverify</cargo.jvmargs>
```

Comment:
I have no idea why we would encourage users to specify `-noverify`: maybe that used to be a thing we encouraged users to do?